### PR TITLE
WRN-12748: Update eslint 8 related modules

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
ESLint v8.0.0 is [released](https://eslint.org/blog/2021/10/eslint-v8.0.0-released)
Updated dependencies related to eslint 8.

### Resolution

- update eslint 8 & related modules
- add `PropertyDefinition` statement to handling `ClassProperty` statement that was never used in eslint 8

### Additional Considerations
**※ BEFORE THIS CHANGE, UPDATE & RELEASE WRN-12124 FIRST.**

### Links
WRN-12748

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "eslint": "^7.32.0",
+    "eslint": "^8.3.0",
```
Comment (MikyungKim) [inline package.json:42]: Let's update the eslint to ^8.11.0 through the PRs.

Comment (hong6316) [reply package.json]: OK

File: package.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "eslint": "^7.32.0",
-    "mocha": "^9.1.3"
+    "eslint": "^8.11.0",
```
Comment (MikyungKim) [inline package.json:42]:
```suggestion
    "eslint": "^8.12.0",
```

File: package.json [hunk 3]
```diff
@@ -22,10 +22,10 @@
   "dependencies": {
     "doctrine": "^3.0.0",
     "jsx-ast-utils": "^3.2.1",
```
Comment (MikyungKim) [inline package.json:24]:
```suggestion
    "jsx-ast-utils": "^3.2.2",
```

File: package.json [hunk 4]
```diff
@@ -22,10 +22,10 @@
...
-    "minimatch": "^3.0.4"
+    "minimatch": "^5.0.1"
   },
   "peerDependencies": {
-    "eslint": ">=7.0.0"
+    "eslint": ">=8.0.0"
   },
   "engines": {
     "node": ">=0.10"
```
Comment (MikyungKim) [inline package.json:31]:
```suggestion
    "node": ">=10.16.0"
```

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**